addons/website/controllers/main.py

```diff
@@ -369,6 +369,12 @@ def google_console_search(self, key, **kwargs):
 
         return request.make_response("google-site-verification: %s" % request.website.google_search_console)
 
+    @http.route('/website/google_maps_api_key', type='json', auth='public', website=True)
+    def google_maps_api_key(self):
```

Comment:
can't you do it in rpc with a classic read on website ? 
To avoid a public controller that does nothing?